spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, :type => :model do
+
+  it 'should have a valid factory' do
+    expect(build_stubbed :admin_user).to be_valid
+  end
+
+  it { is_expected.to validate_presence_of :email }
+  it { is_expected.to validate_presence_of :password }
+  it { is_expected.not_to validate_presence_of :password_confirmation }
+  it { is_expected.to validate_uniqueness_of :email }
+  it { is_expected.not_to validate_uniqueness_of :password }
+  it { is_expected.not_to validate_uniqueness_of :password_confirmation }
```

Comment:
Yes, I thought of that. But I do it anyways thinking that may be in the future I will have to get rid of them.